mantidproject/mantid
# Refactor isis & sns data archives to use internethelper

# To test

This needs to be tested at ISIS (develop tests done here) and SNS
1. startup mantiplot
2. tick the search archives box in the data paths dialog, or first use dialog
3. make sure your facility is selected
4. Load a file that is not on a local path eg MAR11067
5. If the file still loads then things are good

Ticket http://trac.mantidproject.org/mantid/ticket/11067
